src/components/Table/Table.jsx

```diff
@@ -0,0 +1,350 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import CaretIcon from '../Icon/CaretIcon/CaretIcon';
+
+const boundClassNames = lucidClassNames.bind('&-Table');
+
+const {
+	object,
+	string,
+	bool
+} = React.PropTypes;
+
+/**
+ * `Thead` renders <thead>.
+ *
+ * Any child `<Tr>` will have `isHeader` set to `true` unless otherwise specified.
+ */
+const Thead = React.createClass(createLucidComponentDefinition({
+	render() {
+		const {
+			children
+		} = this.props;
+
+		return (
+			<thead {...this.props} className={boundClassNames('&-thead', this.props.className)}>
+				{React.Children.map(children, (childElement) => React.createElement(childElement.type, {isHeader: true, ...childElement.props}))}
+			</thead>
+		);
+	}
+}));
```

Comment:
There's no technical reason for it, though? Other than for those reasons this could have just been an arrow function component? Just wanted to make sure there wasn't some benefit I was missing out on. :)